daemon/cmd/endpoint.go

```diff
@@ -709,6 +709,14 @@ func (d *Daemon) DeleteEndpoint(id string) (int, error) {
 	} else if err = endpoint.APICanModify(ep); err != nil {
 		return 0, api.Error(DeleteEndpointIDInvalidCode, err)
 	} else {
+		containerID := ep.GetShortContainerID()
+		if containerID != "" {
+			log.WithField("containerID", containerID).Info("Delete endpoint request")
+		} else {
+			// if the containerID does not exist, then the IP address is inserted in the log
+			log.WithField("IPv4", ep.GetIPv4Address()).WithField("IPv6", ep.GetIPv6Address()).Info(
```

Comment:
same as above, IPv4 -> logfields.IPv4, IPv6 -> logfields.IPv6.

Reply:
This is a good idea. I'll do it.